### PR TITLE
added NAS page

### PR DESCRIPTION
**Pull request by sargonas**

### CODE REVIEW

File: docs/admins/truenas.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#  TrueNAS
+
+## General
+----
+We have a NAS unit running TrueNAS at http://vault.synshop.net (http://10.0.40.30). Any member who requires some storage space for project related storage can request an account to be created for them.
```
Comment (mrjones-plip) [inline docs/admins/truenas.md:5]:
```suggestion
We have a NAS unit running [TrueNAS](https://www.truenas.com/) at [vault.synshop.net](http://vault.synshop.net) which is only accessible on the shop LAN and not accessible on the internet. Any member who requires some storage space for project related storage can request an account to be created for them.
```

File: docs/admins/truenas.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+- The stash folder is at the same level.
+- Additional folders can be created as needed.
+
+- There are two user groups: Members and Admins. Folders can be given Members, Admin, or selected individual permissions as needed. More groups can be created if required but is not advised.
+
+## Administration
+----
+To add a new user to the system, you need to log into the web interface (http://vault.synshop.net), which is only possible with the root user. (Credentials are in the administrative keepass).
+To add a user:
+1. Go to Accounts>User in the left hand menu
```
Comment (mrjones-plip) [inline docs/admins/truenas.md:19]: without a break here, the numbers don't render as an `<ol>`:

```suggestion
To add a user:

1. Go to Accounts>User in the left hand menu
```

File: docs/admins/truenas.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+----
+We have a NAS unit running TrueNAS at http://vault.synshop.net (http://10.0.40.30). Any member who requires some storage space for project related storage can request an account to be created for them.
+
+- There is also a folder called "stash" that is a public, general temporary space that is mounted as a shared storage by all shop computers to quickly and easily move files between devices.
+
+- The Vault pool has a folder called "members" under which each user with an active account on the NAS has a personal folder.
+- The stash folder is at the same level.
+- Additional folders can be created as needed.
+
+- There are two user groups: Members and Admins. Folders can be given Members, Admin, or selected individual permissions as needed. More groups can be created if required but is not advised.
```
Comment (mrjones-plip) [inline docs/admins/truenas.md:13]: snug up bullet points so the don't render with spaces in final doc:

```suggestion

- There is also a folder called "stash" that is a public, general temporary space that is mounted as a shared storage by all shop computers to quickly and easily move files between devices.
- The Vault pool has a folder called "members" under which each user with an active account on the NAS has a personal folder.
- The stash folder is at the same level.
- Additional folders can be created as needed.
- There are two user groups: Members and Admins. Folders can be given Members, Admin, or selected individual permissions as needed. More groups can be created if required but is not advised.
```

File: docs/admins/truenas.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## Administration
+----
+To add a new user to the system, you need to log into the web interface (http://vault.synshop.net), which is only possible with the root user. (Credentials are in the administrative keepass).
```
Comment (mrjones-plip) [inline docs/admins/truenas.md:17]: > administrative keepass

Oh!  I would have expected it in the Systems keepass instead!  Are there other web/system logins in there?  We should decided which is best and consolidate.

also, distilling down some of the wording:

```suggestion
To add a new user to the system, you need to log into the [web interface](http://vault.synshop.net) as `root` user. Credentials for `root` in the administrative keepass in Google Drive under `TrueNAS`.
```

File: docs/admins/truenas.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+3. Fill in the actual members name, and their chosen username (default to their Discord/known-as identity when possible)
+4. The email address is optional, but preferred. Use the member's normal contact email for future compatibility.
+5. Set a password for the member, or let them set one themselves. Note that they can only change it by SSHing into the NAS.
+6. Leave the userID to the programatically assigned one.
+7. Uncheck "new primary group" and set their group to the existing "members" group.
+8. select /mnt/vault/members from the menu as their default home directory location. (the system will auto create a folder for them with their username)
+9. Submit!
+
+The unit is a Rackable Systems with 4x2TB sata drives, with a SAS cable to a 16 drive unit.
+- Drive 0 (far left) in the control unit is the OS drive at 2TB
```
Comment (mrjones-plip) [inline docs/admins/truenas.md:30]: Can you add a photo with some labels?  A picture is worth a thousand words after all ;)

need a space to render the bullet points:
```suggestion
The unit is a Rackable Systems with 4x2TB SATA drives, with a SAS cable to a 16 drive unit.

- Drive 0 (far left) in the control unit is the OS drive at 2TB
```

File: docs/admins/truenas.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+- Drives 2-4 are in a RAIDz pool called "Plex" for media storage.
+- Drives 5-7 are currently unused 2TB drives but will slowly be used to phase in larger capacity drives.
+- Drives 8-20 are 3Tb drives in a RAIDz3 pool called "Vault". This is the main NAS storage.
+
+There are two network devices. LAN 1 is static assigned to 10.0.40.30. TrueNAS takes a hardline stance on never having 2 NICs on the same subnet, and has designed many elements of the OS to prevent this, so LAN 2 is not used at this time.
+
+Future to do:
+- [ ] Setup second nic on the local network on an appropriately different subnet
+- [ ] Bind both nics into one for higher throughput and reliability
+- [ ] Eventually replace user management with centralized system for the whole shop
```
Comment (mrjones-plip) [inline docs/admins/truenas.md:40]: break to render bullets, and mkdocs doesn't do check marks? dunno, I tried and failed. Removing for now:

```suggestion
Future to do:

-  Setup second nic on the local network on an appropriately different subnet
-  Bind both nics into one for higher throughput and reliability
-  Eventually replace user management with centralized system for the whole shop
```

**Review by mrjones-plip — CHANGES_REQUESTED** [review]
@sargonas  - you ROCK!!  One our best PRs, thank you!

minor formatting changes, but mostly good!  We also need a DNS entry I think for `vault`?  Let me know - I'm happy to help!

File: docs/admins/truenas.md (hunk 6 above)
Comment (sargonas) [reply docs/admins/truenas.md:40]: Does our RTFM markdown flavor not support checkboxes?

Comment (mrjones-plip) [reply docs/admins/truenas.md:40]: :shrug: :

> dunno, I tried and failed. Removing for now

**Review by mrjones-plip — APPROVED** [review]
approving to unblock, but would love an image with labels so folks know which drive is which and such!

here's a raw image to work with if you wanna:

rack syn shop

(i also don't know if you can merge?  If not, lemme know)

**Comment by sargonas** [comment]
Nope, I can't merge, hence the PR... feel free to add me tt the repo though. (plus I'm a GH purist who doesn't like merging my own PRs without a secondary review but happy to do so so long as someone else reviewed it.)

**Pull request merged**